jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -126,8 +129,11 @@ def on_mouse_or_key_event(self, data):
                     and x < image.shape[1] - 0.5 and y < image.shape[0] - 0.5
                     and hasattr(active_layer, 'attribute')):
                 attribute = active_layer.attribute
-                value = image.get_data(attribute)[int(round(y)), int(round(x))]
-                unit = image.get_component(attribute).units
+                value = image.get_data(attribute, view=[np.atleast_1d(y), np.atleast_1d(x)])[0]
```

Comment:
This ensures that we only ever access the relevant pixel and don't accidentally load all the data then slice it.